bio/flye/wrapper.py

```
@@ -0,0 +1,113 @@
+        f"Must be one of: {', '.join(seq_types.keys())}"
+    )
+
+if param_seq_type in seq_types:
+    input_arg += f" {seq_types[param_seq_type]}"
+else:
+    raise ValueError(
+        f"Invalid seq_type: {param_seq_type}."
+        f"Must be one of: {', '.join(seq_types.keys())}"
+    )
+
+reads = snakemake.input[0]
```

Ruff (0.15.11)

[error] Undefined name `snakemake`

(F821)